app/mailers/updated_terms_mailer.rb

```diff
@@ -0,0 +1,11 @@
+class UpdatedTermsMailer < ApplicationMailer
+  layout nil
+
+  def notify(recipient, name)
+    @name = name
+    @accept_terms_url = "https://meedan.com/legal/terms-of-service"
```

Comment:
Question: Why this is hard-coded instead of reading from the configuration key `tos_url` (https://github.com/meedan/check-api/blob/develop/config/config.yml.example#L70)?

Reply:
Fixed